tests/components/blebox/test_cover.py

```diff
@@ -365,17 +368,33 @@ async def test_with_no_stop(gatebox, hass, config):
     assert not supported_features & SUPPORT_STOP
 
 
-@pytest.mark.parametrize("wrapper", ALL_COVER_FIXTURES, indirect=["wrapper"])
-async def test_update_failure(wrapper, hass, config):
+@pytest.mark.parametrize("feature", ALL_COVER_FIXTURES, indirect=["feature"])
+async def test_update_failure(feature, hass, config, caplog):
     """Test that update failures are logged."""
 
-    feature_mock, entity_id = wrapper
+    caplog.set_level(logging.ERROR)
 
+    feature_mock, entity_id = feature
     feature_mock.async_update = AsyncMock(side_effect=blebox_uniapi.error.ClientError)
-    name = feature_mock.full_name
+    await async_setup_entity(hass, config, entity_id)
+
+    assert f"Updating '{feature_mock.full_name}' failed: " in caplog.text
+
+
+@pytest.mark.parametrize("feature", ALL_COVER_FIXTURES, indirect=["feature"])
+async def test_entity_properties(feature, hass, config):
+    """Test that entity properties work."""
+
+    feature_mock, entity_id = feature
+
+    # NOTE: I don't know how to test these other than directly through an entity
```

Comment:
One potential option is to set the mock and then call the generic entity update service to write the state then check hass.states.get(entity_id).state.  I believe you may have to implement `async_update` in the entity.
```
await async_setup_component(hass, "homeassistant", {})
    await hass.services.async_call(
        "homeassistant",
        "update_entity",
        {"entity_id": "cover.XXXX"},
        blocking=True,
    )
```

@MartinHjelmare will probably have a better way to do this though

Reply:
The entity state will return `is_closing` etc if the corresponding property is truthy:
https://github.com/home-assistant/core/blob/c19b222d5a5ab1797d045e6edeaf497bfcbac5ef/homeassistant/components/cover/__init__.py#L184-L197

Reply:
Thanks, that makes a lot of sense now. (I saw that `state` property, but I didn't figure out at first to use it).